### ISSUE TITLE
CRAM Output support

### ISSUE THREAD

**Issue by apeltzer** [nf-core/eager#181]
We should have the possibility to output CRAM files

**Comment by apeltzer** [comment]
Issues with this:

- DeDup can't handle CRAM yet
- DamageProfiler can't handle CRAM yet

**Comment by jfy133** [comment]
@apeltzer I think it makes more sense to close this until fixed upstream ;) and we get more requests. Thoughts?

**Comment by apeltzer** [comment]
Maybe - damageprofiler though can do cram now :)

**Comment by jfy133** [comment]
Review of our tools that accepts CRAM format:

- [x] samtools fastq ☑️
- [x] samtools flagstat ☑️
- [x] hostremoval 🟡 (but probably easily added, as uses pysam, which does)
- [x] samtools view ☑️
- [x] dedup ❌
- [x] picard markDuplicates ☑️
- [x] bedtools ☑️
- [x] damageprofiler ☑️
- [x] mtnucratio ❌
- [x] preseq 🟡 (could just use bed format instead?)
- [x] nuclear contamination ☑️
- [x] qualimap ☑️ (not directly, but @FriederikeHanssen has made a sneaky nf-core module to make it 'supportable')
- [x] PMDTools ☑️ not natively, but uses a pipe from samtools view
- [x] BamUtils ❌
- [x] mapDamage2 🟡 (uses pysam, seemingly without particular version? Maybe work...)
- [x] GATK UnifiedGenotyper ❌
- [x] GatkHaplotypeCaller ☑️ 
- [ ] sequenceTools
- [ ] [Freebayes](https://github.com/freebayes/freebayes/issues/592#issuecomment-770340026)  ❌ 
- [ ] ANGSD ☑️

So actually I guess most tools accept it...

**Comment by jfy133** [comment]
Ping @apeltzer for DeDup/MTNucRatio ;)

**Comment by apeltzer** [comment]
Yeah supported, worst case I just bump the library used and it should work almost out of the box.
